minecraft_servers/marsmc/manifest.json

```
@@ -0,0 +1,18 @@
+{
+  "server_name": "marsmc",
+  "nice_name": "MarsMC",
+  "direct_ip": "marsmc.net",
+   "server_wildcards":[

```

Fix your intendation:
```suggestion
  "server_wildcards": [

```

---

Fixed ✅